pkg/cmd/infra/deployer/deployer.go

```diff
@@ -78,7 +78,7 @@ func NewCommandDeployer(name string) *cobra.Command {
 }
 
 // NewDeployer makes a new Deployer from a kube client.
-func NewDeployer(client kclient.Interface) *Deployer {
+func NewDeployer(client *kclient.Client) *Deployer {
```

Comment:
@smarterclayton? I don't have a feel for the effort involved in exposing pod logs through the k8s client interface. Aware of any work in this area?